src/Compilers/CSharp/Portable/BoundTree/Statement.cs

```diff
@@ -843,6 +873,22 @@ partial class BoundPatternSwitchStatement
         // TODO: this may need its own OperationKind.
         protected override OperationKind StatementKind => OperationKind.None;
 
+        protected override ImmutableArray<IOperation> Children
+        {
+            get
+            {
+                var builder = ImmutableArray.CreateBuilder<IOperation>(this.SwitchSections.Length + 2);
+                builder.Add(this.Expression);
+                foreach (var section in this.SwitchSections)
+                {
+                    builder.Add(section);
+                }
+                builder.Add(this.DefaultLabel);
```

Comment:
>builder.Add(this.DefaultLabel); [](start = 16, length = 31)

It feels like one of the sections refers to this node and this member is just for compiler's convenience.